features/aanspreekvorm.feature

```diff
@@ -0,0 +1,659 @@
+      | naam                        | waarde    |
+      | burgerservicenummer         | 999992934 |
+      | geslachtsaanduiding (04.10) | O         |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                                 | waarde               |
+      | adellijke titel of predicaat (02.20) | <titel of predicaat> |
+      | aanduiding naamgebruik (61.10)       | <naamgebruik>        |
+      En de persoon heeft nooit een actueel of ontbonden huwelijk of partnerschap gehad
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met leeg 'adressering' object
+
+      Voorbeelden:
+      | naamgebruik | titel of predicaat |
+      | E           | JH                 |
+      | P           | JV                 |
+      | V           | R                  |
+      | N           | B                  |
+      | E           | BS                 |
+      | P           | H                  |
+      | V           | HI                 |
+      | N           | G                  |
+      | E           | GI                 |
+      | P           | M                  |
+      | V           | MI                 |
+
+    Abstract Scenario: persoon heeft geen adellijke titel/predicaat en geen (ex-)partner
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde                   |
+      | aanduiding naamgebruik (61.10) | <aanduiding naamgebruik> |
+      En de persoon heeft nooit een actueel of ontbonden huwelijk of partnerschap gehad
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992934                       |
+      | fields              | aanschrijfwijze.aanspreekvorm   |
+      Dan heeft de response een persoon met een leeg 'adressering' object
+
+      Voorbeelden:
+      | aanduiding naamgebruik |
+      | E                      |
+      | P                      |
+      | V                      |
+      | N                      |
+
+    Abstract Scenario: persoon en partner hebben geen adellijke titel/predicaat
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992934 |
+      En de persoon heeft de volgende 'naam' gegevens
+      | naam                           | waarde                   |
```

Comment:
heeft de persoon geen naam en de partner ook niet? Dan illustreer je impliciet twee regels:
1. wanneer de persoon geslachtsnaam is leeg en naamgebruik≠P wordt aanschrijfwijze niet opgenomen
2. wanneer de partner geslachtsnaam is leeg wordt naamgebruik=E gehanteerd

Lijkt me lastig hieruit te halen wanneer je dat niet al weet (dus voor ieder ander dan wij twee)